src/app/review-rating/review-rating.component.scss

```diff
@@ -0,0 +1,104 @@
+.review-rating {
+  top: 0 !important;
+  padding: 0 10px 0 10px;
+
+  .header-detail {
+    display: inline-flex;
+    width: 100%;
+    margin-bottom: 10px;
+    margin-top: 20px;
+
+    .title {
+      margin: auto;
+      padding-left: 10px;
+      margin-left: 0;
+      font-size: 20px;
+      font-weight: normal;
+      color: var(--ion-color-primary);
```

Comment:
Don't need to specify the color as primary, since by default all colors are primary. If you have to specify the color, specify it in the directive, like `color="primary"`, don't add extra CSS for color. 
Same as below